modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc

```diff
@@ -274,7 +277,8 @@ The Instance Metadata Service settings cannot be changed after your cluster is c
 If you specified custom ARN paths when you created the associated account-wide roles, the custom path is automatically detected. The custom path is applied to the cluster-specific Operator roles when you create them in a later step.
 ====
 <6> Optional: Multiple availability zones are recommended for production workloads. The default is a single availability zone.
-<7> Optional: Enable this option if you are using your own AWS KMS key to encrypt the control plane, infrastructure, worker node root volumes, and PVs. Specify the ARN for the KMS key that you added to the account-wide role ARN in the preceding step.
+<7> Optional: You can create a cluster in an existing VPC if you have already created the VPC, and if no other OpenShift cluster is currently using or has previously used that VPC.
```

Comment:
@gdbranco There's a warning in the docs about a problem where if someone deletes a ROSA cluster in a shared VPC, the VPC is deleted as part of cluster deletion, regardless of whether other clusters are in it. If that's been fixed, that's great, but this was the best guess I had for restrictions around that. @tsze-redhat @yuwang-RH Do you know if this is still a problem?

Reply:
Actual text of the existing important note:

To avoid cluster failure issues, do not install a ROSA cluster in an existing VPC created by the previous cluster's ROSA installer. If the cluster that created the VPC during the installation is deleted, the associated installer-created VPC also gets deleted, resulting in the failure of all the clusters installed in the same VPC.

CC @xueli181114 on the above - any idea if this is still an issue?

Reply:
> To avoid cluster failure issues, do not install a ROSA cluster in an existing VPC created by the previous cluster's ROSA installer. If the cluster that created the VPC during the installation is deleted, the associated installer-created VPC also gets deleted, resulting in the failure of all the clusters installed in the same VPC.

This is correct^ installing a cluster into a VPC that was managed as a part of a separate cluster install is not something we want to encourage.

Reply:
Ah, thankyou, sounds like the managed part is the concern - @mjlshen if they manage their own VPC, would multiple ROSA clusters be supported?

Reply:
In the meantime, I'm going to add the relevant warning here and just say that you can install into an existing VPC.

Reply:
@mjlshen But this is for a managed VPC, this feature (security groups) is only for BYO VPC

Reply:
The problem here we need to distinguish the vpc created by customer or cluster. The VPC created by OCM cluster creation will have a tag "red-hat-managed:true". The vpc will be deleted with deletion of cluster, of course, it will cause deletion failure and creation failure. We should highlight this part. And for BYOVPC, customer can re-use the vpc between clusters but it's not recommended. I think we should highlight it too.

Reply:
I've changed this to the following:

```
Optional: You can create a cluster in an existing VPC, or ROSA can create a new VPC to use.
+
[IMPORTANT]
====
To avoid cluster failure issues, do not install a ROSA cluster in an existing VPC created by the previous cluster's ROSA installer. If the cluster that created the VPC during the installation is deleted, the associated installer-created VPC also gets deleted, resulting in the failure of all the clusters installed in the same VPC.
====
```

If there are additional changes to be made here, I recommend creating another jira to handle this issue separately.